build/images.bzl

```diff
@@ -15,11 +15,14 @@
 load("@io_bazel_rules_docker//container:container.bzl", "container_pull")
 
 def define_base_images():
-    ## Use 'static' distroless image for all builds
+    # Use 'static' distroless image for all builds
+    # To get the latest-amd64 digest for gcr.io/distroless/static, assuming
+    # that $GOPATH/bin is in your $PATH, run:
+    # go get github.com/genuinetools/reg && reg digest gcr.io/distroless/static:latest-amd64
```

Comment:
Interesting how we're using amd64 base to build multi-arch images (I checked that the current base is amd64 too).